### PR TITLE
Create navneet17360.json

### PR DESCRIPTION
**Pull request by navneet17360**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/6446"></a>

### CODE REVIEW

File: data/navneet17360.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+	"bio": "Front-End Developer",
+	"links": [{
+			"name": "Follow me on Instagram",
+			"url": "https://www.instagram.com/navneet_anand_17",
+			"icon": "FaInstagram"
+		},
+		{
+			"name": "Follow me on Github",
+			"url":"https://github.com/navneet17360",
+			"icon:"FaGithub"
```
Comment (jaydeepkhatri) [inline data/navneet17360.json:13]: The JSON is not properly formatted, 
You missed to close the quotation for "icon".

```json
"icon": "FaGithb"
```

**Review by jaydeepkhatri — CHANGES_REQUESTED** [review]
Hi @navneet17360 
Thank you for creating a PR at Linkfree.

I found that the JSON is invalid.
Use this tool to verify the Code.
-> https://linkfree.io/playground

Thanks :)

File: data/navneet17360.json (hunk 1 above)
Comment (SaraJaoude) [inline data/navneet17360.json:13]: needs the following corrections:
```suggestion
			"icon": "FaGithub",
```

File: data/navneet17360.json [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+	"bio": "Front-End Developer",
+	"links": [{
+			"name": "Follow me on Instagram",
+			"url": "https://www.instagram.com/navneet_anand_17",
+			"icon": "FaInstagram"
+		},
+		{
+			"name": "Follow me on Github",
+			"url":"https://github.com/navneet17360",
+			"icon": "FaGithub",
```
Comment (SaraJaoude) [inline data/navneet17360.json:13]:
```suggestion
			"icon": "FaGithub"
```

**Review by SaraJaoude — APPROVED** [review]
looks good

**Pull request merged**